examples/pysdl2.py

```diff
@@ -0,0 +1,357 @@
+    cef.Initialize(settings={"windowless_rendering_enabled": True})
+    window_info = cef.WindowInfo()
+    window_info.SetAsOffscreen(0)
+    # Initialise SDL2 for video (add other init constants if you
+    # require other SDL2 functionality e.g. mixer,
+    # TTF, joystick etc.
+    sdl2.SDL_Init(sdl2.SDL_INIT_VIDEO)
+    # Create the window
+    window = sdl2.video.SDL_CreateWindow(
+        'cefpython3 SDL2 Demo',
+        sdl2.video.SDL_WINDOWPOS_UNDEFINED,
+        sdl2.video.SDL_WINDOWPOS_UNDEFINED,
+        width,
+        height,
+        0
+    )
+    # Define default background colour (black in this case)
+    backgroundColour = sdl2.SDL_Color(0, 0, 0)
+    # Create the renderer using hardware acceleration
+    renderer = sdl2.SDL_CreateRenderer(window, -1, sdl2.render.SDL_RENDERER_ACCELERATED)
+    # Set-up the RenderHandler, passing in the SDL2 renderer
+    renderHandler = RenderHandler(renderer, width, height - headerHeight)
+    # Create the browser instance
+    browser = cef.CreateBrowserSync(window_info, url="https://www.google.com/")
+    browser.SetClientHandler(LoadHandler())
+    browser.SetClientHandler(renderHandler)
+    # Must call WasResized at least once to let know CEF that
+    # viewport size is available and that OnPaint may be called.
+    browser.SendFocusEvent(True)
+    browser.WasResized()
+    # Begin the main rendering loop
+    shiftDown = False
+    running = True
+    while running:
+        # Convert SDL2 events into CEF events (where appropriate)
+        events = sdl2.ext.get_events()
+        for event in events:
+            if event.type == sdl2.SDL_QUIT or (event.type == sdl2.SDL_KEYDOWN and event.key.keysym.sym == sdl2.SDLK_ESCAPE):
+                    running = False
+                    break
+            if event.type == sdl2.SDL_MOUSEBUTTONDOWN:
+                if event.button.button == sdl2.SDL_BUTTON_LEFT:
+                    if event.button.y > headerHeight:
+                        # Mouse click triggered in browser region
+                        browser.SendMouseClickEvent(
+                            event.button.x,
+                            event.button.y - headerHeight,
+                            cef.MOUSEBUTTON_LEFT,
+                            False,
+                            1
+                        )
+            elif event.type == sdl2.SDL_MOUSEBUTTONUP:
+                if event.button.button == sdl2.SDL_BUTTON_LEFT:
+                    if event.button.y > headerHeight:
+                        # Mouse click triggered in browser region
+                        browser.SendMouseClickEvent(
+                            event.button.x,
+                            event.button.y - headerHeight,
+                            cef.MOUSEBUTTON_LEFT,
+                            True,
+                            1
+                        )
+            elif event.type == sdl2.SDL_MOUSEMOTION:
+                if event.button.y > headerHeight:
```

Comment:
Why this condition? Why aren't you sending all mouse move events? When I hover over a link at Google I don't see link being underlined, because mouse move event is not being sent.